hr_attendance_autoclose/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2018 Eficent Business and IT Consulting Services, S.L.
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    'name': 'HR Attendance Auto Close',
+    'version': '11.0.1.0.0',
+    'category': 'Human Resources',
+    'summary': 'Close stale Attendances',
+    'website': 'https://github.com/OCA/hr',
+    'author': 'Odoo Community Association (OCA)',
+    'license': 'AGPL-3',
+    'installable': True,
+    'depends': [
+        'hr_attendance',
```

Comment:
```suggestion
```

Its already on https://github.com/OCA/hr/blob/11.0/hr_attendance_reason/__manifest__.py#L15

Reply:
True, I just changed it.